doc/conf.py

```diff
@@ -53,6 +53,8 @@
 automodapi_toctreedirnm = "code/api"
 automodsumm_inherited_members = True
 
+bibtex_bibfiles = ["bibfile.bib"]
```

Comment:
Let's add a comment here on why this line is required.

Reply:
Done.